src/debugpy/adapter/clients.py

```diff
@@ -701,6 +702,24 @@ def disconnect_request(self, request):
     def disconnect(self):
         super().disconnect()
 
+    def report_sockets(self):
+        sockets = [
```

Comment:
I'm not familiar with this list initialization syntax. Is there supposed to be a comma after the dict? Does the code afterwards resolve to a list and it gets stuffed into the sockets parent list?